netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1546,6 +1546,16 @@ net_ebpf_extension_sock_addr_redirect_connection_classify(
         goto Exit;
     }
 
+    if (filter_context->base.client_detached) {
```

Comment:
This trace should be present on all classify IMO with verbose level.

Reply:
ugh. missed this.  will update

Reply:
fixed.